src/server/enterprise/cmds/cmds.go

```diff
@@ -45,6 +45,7 @@ func ActivateCmd() *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("could not connect: %s", err.Error())
 			}
+			defer c.Close()
```

Comment:
This is purely for my own understanding, but given that these commands usually issue a single RPC and then end the process, is there anything that would be leaked if we didn't call close (a file or socket or something?). I think the answer is no, and that these are good hygiene but not strictly necessary, but I'm not really sure